freeipa.spec.in

```diff
@@ -843,20 +856,25 @@ pushd %{_builddir}/freeipa-%{version}-python3
 (cd ipalib && %make_install)
 (cd ipaplatform && %make_install)
 (cd ipapython && %make_install)
+%if ! %{ONLY_CLIENT}
```

Comment:
Why is ```--without-ipatests``` required at all when you don't install ipatests in ONLY_CLIENT builds any way? This shows that packaging is easily possible without the new argument.

CC @simo5